app/assets/stylesheets/omni_nav/omni_nav_v2/omni-nav.scss

```diff
@@ -588,6 +589,7 @@ html.omni-nav-v2 {
           & > a {
             text-transform: uppercase;
             font-size: 15px;
+            white-space: nowrap;
```

Comment:
Added this `white-space: nowrap` rule to the 2 areas where users reported the weirdly wrapped text on zoomed-in screens.